packages/core/admin/admin/src/components/GuidedTour/init.js

```diff
@@ -26,8 +26,8 @@ const init = initialState => {
     persistStateToLocaleStorage.addCurrentStep(null);
   }
 
-  if (skippedLocaleStorage) {
-    set(copyInitialState, 'isSkipped', true);
+  if (skippedLocaleStorage !== null) {
+    set(copyInitialState, 'isSkipped', skippedLocaleStorage);
```

Comment:
Don't you need to parse the value?

Reply:
already parsed with `const skippedLocaleStorage = persistStateToLocaleStorage.get(SKIPPED)`
```
  get: item => {
    return parse(localStorage.getItem(item));
  },
```